src/redux/reducers/stock/stockReducer.test.js

```diff
@@ -57,6 +60,7 @@ describe('Async action creator', () => {
   let store;
 
   beforeEach(() => {
+    render(<SnackbarProvider><SnackbarUtilsConfigurator /></SnackbarProvider>);
```

Comment:
Instead of that we should test notification methods calls inside `getTicker()` method to be sure that each scenario inside this method will trigger proper notification, right? Having spies we have mock for free. So instead of that we can create spy on that object methods like:
```
import { snackActions } from '<path>'
const toastSpy = jest.spyOn(snackActions, 'toast')

// in success/fail actions assertion test check that things out

expect(toastSpy).toHaveBeenCalledWith('successOrSomething')
```